docs/4.1/components/card.md

```diff
@@ -47,6 +47,19 @@ The building block of a card is the `.card-body`. Use it whenever you need a pad
 {% endcapture %}
 {% include example.html content=example %}
 
+### Body
+
+To have smaller version of our card-body, we provide a card-body-slim class.
+
+{% capture example %}
+<div class="card">
+  <div class="card-body-slim">
```

Comment:
Actually the bootstrap way of adding a different version of a component would be `card-body card-body-slim` we should stick with this schema